CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This file is to automatically add code owners in PR reviews
-* @hyperledger-updates-maintaineres
+* @hyperledger-tooling/hyperledger-updates-maintaineres
```

Comment:
_maintainers_ not _maintaineres_

Reply:
I copied this text from the GitHub

![image](https://user-images.githubusercontent.com/27664223/123678935-8d119100-d864-11eb-99e8-ac724af863db.png)

Reply:
(I renamed it to fix the typo)

Reply:
Fixed, thanks